docs/rfcs/XeGPU.md

```diff
@@ -55,7 +55,14 @@ Attribute `mode` indicates whether the XeGPU operation is working under “Vecto
 
 Any XeGPU operation working at VC mode needs to explicitly declare this attribute. The default mode is SIMT mode. When VC mode is on, the sg_map attribute should not be presented in the associated tensor_desc.
 
-create_nd also accepts a memref as input instead of a memory address, shapes, and sizes.
+create_nd_tdesc creates a tensor descriptor that covers an array of 2D subtensor. The size being covered by the tensor_desc is multiplied with the array_length along the innermost dimension. The subtensor being created in the example below covers 8x32xbf16. 
```

Comment:
I'm confused. Shouldn't it be along the outer dimension? Expanding the inner dimension would make later extract step harder. Maybe that is the way HW works.

Reply:
Yes. It is how HW works. It effectively does a blocking.